README.md

```diff
@@ -18,40 +18,99 @@ Some of these are tricky!  Don't trust your first instinct.
 
```

Comment:
Nice job throughout! I like that you included the console results and your thoughts about any differences between expected and actual.

One procedural note: please make sure that you're working in a personal directory inside the repo, otherwise if your homework gets merged in it will overwrite the initial version.